modules/overrides/userconfigurable/api/http.go

```diff
@@ -82,13 +85,18 @@ func (a *UserConfigOverridesAPI) PostHandler(w http.ResponseWriter, r *http.Requ
 		return
 	}
 
+	skipConflictingOverridesCheck := false
+	if value, ok := r.URL.Query()[queryParamSkipConflictingOverridesCheck]; ok && len(value) > 0 {
+		skipConflictingOverridesCheck, _ = strconv.ParseBool(value[0])
```

Comment:
Do we want to handle the error?

Reply:
I opted to silently 'fail' but just returning an error isn't a lot of work + feels nicer